src/Controller/CkeditorAdminController.php

```diff
@@ -187,12 +192,19 @@ private function setFormTheme(FormView $formView, $theme)
         $twig = $this->get('twig');
 
         // BC for Symfony < 3.2 where this runtime does not exists
-        if (!method_exists('Symfony\Bridge\Twig\AppVariable', 'getToken')) {
-            $twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')
-                ->renderer->setTheme($formView, $theme);
+        if (!method_exists(AppVariable::class, 'getToken')) {
+            $twig->getExtension(FormExtension::class)->renderer->setTheme($formView, $theme);
 
             return;
         }
-        $twig->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer')->setTheme($formView, $theme);
+
+        // BC for Symfony < 3.4 where runtime should be TwigRenderer
+        if (!method_exists(DebugCommand::class, 'getLoaderPaths')) {
+            $twig->getRuntime(TwigRenderer::class)->setTheme($formView, $theme);
+
+            return;
+        }
```

Comment:
Since when does `FormRenderer` exist?

Reply:
Since 3.0 I think, but until 3.4 it was not possible to get it using `getRuntime`. This is a copy paste of the solution applied to AdminBundle, and MediaBundle

Reply:
http://api.symfony.com/2.3/Symfony/Component/Form/FormRenderer.html

Reply:
Yep, it might be since a lot of time, but until 3.4 it was not available to get it from runtime :)

Reply:
Ok then